src/celeritas/field/FieldDriver.hh

```diff
@@ -348,9 +342,6 @@ FieldDriver<StepperT>::one_good_step(real_type step, OdeState const& state) cons
         }
     } while (!succeeded && --remaining_steps > 0);
 
-    // TODO: loop check and handle rare cases if happen
-    CELER_ASSERT(succeeded);
```

Comment:
Removing these assertions (succeeded with an incomplete integration) cause problems in the upstream FieldPropagator.  So, propose to revert, but increase  the default max_nsteps from 100 to 500 for temporarily until all remaining problems of very low energy particle are understood and fixed.

Reply:
Uh oh. What are the problems upstream?

Reply:
The first hit is at this [line](https://github.com/sethrj/celeritas/blob/619-field-driver/src/celeritas/field/FieldPropagator.hh#L231) and then cascade the problem to further at t[he end condition](https://github.com/sethrj/celeritas/blob/619-field-driver/src/celeritas/field/FieldPropagator.hh#L285).